SCVersion.txt

```diff
@@ -4,7 +4,7 @@
 set(SC_VERSION_MAJOR 3)
 set(SC_VERSION_MINOR 11)
 set(SC_VERSION_PATCH 0)
-set(SC_VERSION_TWEAK "-beta1")
+set(SC_VERSION_TWEAK "-RC1")
```

Comment:
this really should be `-rc1` to match with https://github.com/supercollider/supercollider/commit/5bc415abd73649df0a3820e9e033d456917f0b0c, https://github.com/supercollider/supercollider/commit/059ede7c6beafa6bf5e67d0e4d3afb23b5180224, https://github.com/supercollider/supercollider/commit/9bec036558a5dbdffcd2bd4aec3de49800a80157, etc.